src/Console/Commands/MvcContractsUpdate.php

```diff
@@ -54,7 +55,7 @@ public function handle(DocumentSignInterface $documentSign)
                     $contract->status = Contract::STATUS_DENIED;
                     break;
                 case 'si':
-                    $contract->signed_at = Carbon::now();
+                    $contract->signed_at = Carbon::createFromTimestamp(strtotime($signRequest['signed_on']), 'Europe/Amsterdam');
```

Comment:
modify contracts:

remove signed_at 
add JSON field signers.

fill json signers like this:
`[{email: 'signer1@sign.nl', signed_at: ''}, {email: 'signer2@sign.nl', signed_at: '2020-01-01 00:00:00'}]`

Notice you cannot use the signed_at date mutator anymore use format('Y-m-d h:i:s') <- i think this is correct to generate a string.